src/settings/StaffSlips/StaffSlipDetail.js

```diff
@@ -1,135 +1,94 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import HtmlToReact, { Parser } from 'html-to-react';
-import {
-  FormattedMessage,
-  injectIntl,
-} from 'react-intl';
+import { FormattedMessage } from 'react-intl';
 
+import { stripesShape } from '@folio/stripes/core';
 import {
-  Button,
   Col,
-  KeyValue,
   Row,
+  ExpandAllButton,
+  Accordion,
+  AccordionSet,
 } from '@folio/stripes/components';
-import { PreviewModal, tokensReducer } from '@folio/stripes-template-editor';
 
-import getTokens from './tokens';
-import css from './StaffSlipDetail.css';
+import { Metadata } from '../components';
+import {
+  StaffSlipAboutSection,
+  StaffSlipTemplateContentSection,
+} from './components/ViewSections';
 
 class StaffSlipDetail extends React.Component {
   static propTypes = {
     initialValues: PropTypes.object.isRequired,
-    intl: PropTypes.object.isRequired,
+    stripes: stripesShape.isRequired,
   };
 
   constructor(props) {
     super(props);
 
-    const processNodeDefinitions = new HtmlToReact.ProcessNodeDefinitions(React);
-    this.parser = new Parser();
-
-    this.rules = [
-      {
-        shouldProcessNode: () => true,
-        processNode: processNodeDefinitions.processDefaultNode,
-      },
-    ];
-
     this.state = {
-      openPreview: false,
+      sections: {
+        generalInformation: true,
+        templateContent: true
+      }
     };
   }
 
-  openPreviewDialog = () => {
-    this.setState({ openPreview: true });
+  handleSectionToggle = ({ id }) => {
+    this.setState(({ sections }) => {
+      sections[id] = !sections[id];
+      return { sections };
```

Comment:
Could you please add an empty line before return?

Reply:
done